### ISSUE TITLE
magit-log-show-margin -> t not respected

### ISSUE THREAD

**Issue by michael-heerdegen** [magit/magit#2325]
Hi,

(I use the current Melpa version)

a quite new problem.  I have magit-log-show-margin -> t (the default) but when using l l, the margin is not visible in the log buffer.  M-x magit-toggle-margin shows it.

The problem seems to be that things are called in this order:
```
1 -> (magit-log-mode)
| 2 -> (magit-set-buffer-margin t)
| 2 <- magit-set-buffer-margin: nil
1 <- magit-log-mode: nil
======================================================================
1 -> (magit-display-buffer #<buffer *magit-log: gnu-emacs>)
1 <- magit-display-buffer: nil
```

`magit-set-buffer-margin' can't set the window margins because the window showing the log is not yet existent at that moment.

Thanks in advance.

**Comment by tarsius** [comment]
Fixed in 0ace20571f590899c8f09eeb6c6e87bbdbb893a8 and 3f98edbcd04e2a4d55348ff689806603a7db40cc.

**Comment by tarsius** [comment]
Thanks.

**Comment by michael-heerdegen** [comment]
Jonas Bernoulli <notifications@github.com> writes:

> Fixed in 0ace205 and 3f98edb.

Thanks!
